tests/http/test_bad_request_with_response_id.py

```diff
@@ -0,0 +1,29 @@
+from bunq.sdk.exception import BadRequestException
+from bunq.sdk.model.generated.endpoint import UserPerson
+from tests.bunq_test import BunqSdkTestCase
+
+
+class TestPagination(BunqSdkTestCase):
+    """
+    Tests if the response id from a failed request can be retrieved
```

Comment:
Docstring should start on the same line as the opening `"""`

Reply:
This is also how we do it 😁 we ignore pep8 's suggestion to begin doctoring right behind `"""
`